src/app/cmdoptions.cpp

```diff
@@ -366,11 +366,11 @@ QStringList QBtCommandLineParameters::paramList() const
     if (!savePath.isEmpty())
         result.append(QString("@savePath=%1").arg(savePath));
 
-    if (addPaused == TriStateBool::True) {
-        result.append(QLatin1String("@addPaused=1"));
-    }
-    else if (addPaused == TriStateBool::False) {
-        result.append(QLatin1String("@addPaused=0"));
+    if (addPaused) {
```

Comment:
use `operator==`

Reply:
There is no operator== (or any comparators) in boost's version of optional.

Reply:
According to [this gotcha](http://www.boost.org/doc/libs/1_66_0/libs/optional/doc/html/boost_optional/tutorial/gotchas.html) you can do:
```c++
if (addPaused == true)
    result.append(QLatin1String("@addPaused=1"));
else if (addPaused == false)
    result.append(QLatin1String("@addPaused=0"));
```

Its simpler. But will it confuse people? Maybe.
I don't have strong preference either way. Do you guys have any argument for or against one or the other?

Reply:
I prefer @sledgehammer999 variant.

Reply:
Done.